### PR TITLE
#36 List Tasks

### LINKED ISSUE #36: List tasks.
## List tasks.
This issue is part of the Epic #35 

### Objective
Create a module to retrieve tasks across all the projects from the Tickspot API.

Documentation: 
- [GET /tasks.json](https://github.com/tick/tick-api/blob/master/sections/tasks.md#get-tasks)

### CheckList
- [ ] Create a method to list closed tasks.
- [ ] Create tests.

### PR DESCRIPTION
**Pull request by MrJuanGaviriaK**
Closes #36
 
## List tasks.
This issue is part of the Epic #35 

### Objective
Create a module to retrieve tasks across all the projects from the Tickspot API.

### CheckList
- [x] Create a method to list opened tasks across all projects.
- [x] Create tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,6 +388,64 @@ const result = await client.tasks.listClosed(callback);
...
+  }
+  ...
+]
+```
+
+Optionally, You can send a callback to perform an action on the response data. e.g:
+
+```javascript
+const callback = (responseData) => {
+  console.log(responseData.map((task) => {
```
Comment (brayvasq) [inline README.md:420]: @MrJuanGaviriaK I think it's better for this example if we return the mapped response for this callback.

File: src/v2/resources/tasks.js [hunk 2]
```diff
@@ -16,6 +16,19 @@ class Tasks extends BaseResource {
...
   }
+
+  /**
+   * This method will return all opened tasks across all projects.
+   * @param {callback} responseCallback
+   *    is an optional function to perform a process over the response data.
+   *
+   * @return {Array} list of all opened tasks across all projects..
+   */
+
```
Comment (brayvasq) [inline src/v2/resources/tasks.js:27]: @MrJuanGaviriaK Can you remove this breakline please :pray: ? Thanks!.

**Review by brayvasq — COMMENTED** [review]
@MrJuanGaviriaK Good Job! :muscle:. I left a few comments.

**Comment by MrJuanGaviriaK** [comment]
Hi @brayvasq 

Done!

Thanks

**Review by brayvasq — APPROVED** [review]

File: src/v2/resources/tasks.js [hunk 3]
```diff
@@ -16,6 +16,18 @@ class Tasks extends BaseResource {
     const URL = `${this.baseURL}/tasks/closed.json`;
     return this.makeRequest({ URL, method: 'get', responseCallback });
   }
+
+  /**
+   * This method will return all opened tasks across all projects.
+   * @param {callback} responseCallback
+   *    is an optional function to perform a process over the response data.
+   *
+   * @return {Array} list of all opened tasks across all projects..
```
Comment (JoseCastroA) [inline src/v2/resources/tasks.js:25]: @MrJuanGaviriaK I little misspelled here: double end dot.

**Review by JoseCastroA — CHANGES_REQUESTED** [review]

File: src/v2/resources/tasks.js (hunk 3 above)
Comment (MrJuanGaviriaK) [reply src/v2/resources/tasks.js]: @JoseCastroA  done! Thanks.

**Pull request merged**